package.json

```diff
@@ -174,6 +174,7 @@
     "@types/react-dom": "17.0.15",
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27770&commitId=a056f0ea3232a1773017ff3be676abfd04c25333&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1258 screenshots unchanged, 0 screenshots with different dimensions and 2 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|List.List Example Selectable Teams.Focuses item.chromium.png|117| Changed|
|List.List Example Selectable Teams.chromium.png|117| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>